src/flatc.cpp

```diff
@@ -167,6 +177,7 @@ int main(int argc, const char *argv[]) {
   bool print_make_rules = false;
   bool raw_binary = false;
   bool schema_binary = false;
+  std::map<int, bool> grpc_enabled;
```

Comment:
Why a map? just do the same as `generator_enabled`.
Actually, `grpc_enabled` can be a single bool, as it simply works for any language that has `generator_enabled`